src/routes/curation-team/+page.svelte

```diff
@@ -30,6 +30,7 @@
     display: grid;
     grid-template-columns: repeat(1, 1fr);
     gap: 1.25rem;
+    word-wrap: break-word;
```

Comment:
This seems good to me. I'd likely do the same.
I did notice the problem doesn't reproduce on Firefox, but I could reproduce it on Chrome, which is maybe why it went unnoticed as well.